docs/source/release_notes.rst

```diff
@@ -3,34 +3,37 @@
 Release Notes
 -------------
 
-.. Future Release
-  ==============
+Future Release
+==============
     * Enhancements
     * Fixes
+        * Fix compatibility issues with featuretools (:pr:`41`)
     * Changes
+        * Rename `normalize_entity` to `normalize_entityset` (:pr:`41`)
```

Comment:
Since this is a breaking change to the API, we typically have been including a Breaking Change note in the release notes. Here is an example: https://github.com/alteryx/woodwork/blob/main/docs/source/release_notes.rst#breaking-changes